src/unsupported/pbs-mailer/README.md

```diff
@@ -0,0 +1,30 @@
+### pbs-mailer
+
+OpenPBS can easily send a huge amount of emails to one user/email. pbs-mailer is a service for an aggregation of emails send by OpenPBS to the same user/email. The first email is send immediately and subsequent emails are sent after configurable time. The emails are squashed into one email.
```

Comment:
emails "sent" by OpenPBS
first email is "sent" immediately
after a configurable time "period".